pkg/framework/aries/api/didcreator/didcreator.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package didcreator
+
+import (
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+)
+
+// Creator allows for creation of DID document
+type Creator interface {
```

Comment:
Question: What about to storage and get DID part inside wallet?

Reply:
Another PR #620 that is currently in progress will expose peer storage functionality so the sequence of calls will be Create followed by Store. This way we can store 'my' and 'their' dids in the same storage. Later on we can probably consolidate creator, storage and resolver  in the same peer method (VDR). This will have to be discussed and handled separately.